api_test.go

```diff
@@ -119,6 +120,26 @@ func TestNetAPI(t *testing.T) {
 		t.Errorf("mismatching HashLookup responses:\ngot  %+v\nwant %+v", gotResp, wantResp)
 	}
 
+	// Test cancelled Context returns an error.
+	wantReq = &pb.FindFullHashesRequest{ThreatInfo: &pb.ThreatInfo{
+		ThreatEntries:    []*pb.ThreatEntry{{Hash: []byte("aaaa")}, {Hash: []byte("bbbbb")}, {Hash: []byte("cccccc")}},
+		ThreatTypes:      []pb.ThreatType{1, 2, 3},
+		PlatformTypes:    []pb.PlatformType{4, 5, 6},
+		ThreatEntryTypes: []pb.ThreatEntryType{7, 8, 9},
+	}}
+	wantResp = &pb.FindFullHashesResponse{Matches: []*pb.ThreatMatch{
+		{ThreatType: 0, PlatformType: 1, ThreatEntryType: 2, Threat: &pb.ThreatEntry{Hash: []byte("abcd")}},
+		{ThreatType: 1, PlatformType: 2, ThreatEntryType: 3, Threat: &pb.ThreatEntry{Hash: []byte("efgh")}},
+		{ThreatType: 2, PlatformType: 3, ThreatEntryType: 4, Threat: &pb.ThreatEntry{Hash: []byte("ijkl")}},
+	}}
+	gotReq = &pb.FindFullHashesRequest{}
+	ctx, cancel := context.WithCancel(context.Background())
+	cancel()
+	_, err = api.HashLookup(ctx, wantReq.(*pb.FindFullHashesRequest))
+	if err == nil {
+		t.Errorf("unexpected HashLookup success, wanted http request cancelled")
```

Comment:
nit: Go style uses one L for canceled. Also, capitalize acronyms: HTTP

Reply:
I don't think that's the first time I've received the cancelled/canceled comment. And it probably wont be the last.